<launch>
  <arg name="model" default="RS128" />
  <arg name="device_ip" default="192.168.1.200"/>
  <arg name="msop_port" default="6699" />
  <arg name="difop_port" default="7788" />
  <arg name="lidar_param_path" default="$(find rslidar_pointcloud)/data/rs_lidar_128/"/>

  <node  name="rslidar_node" pkg="rslidar_driver" type="rslidar_node" output="screen" >
    <param name="model" value="$(arg model)"/>
    <param name="device_ip" value="$(arg device_ip)" />
    <param name="msop_port" value="$(arg msop_port)" />
    <param name="difop_port" value="$(arg difop_port)"/>
    <param name="cut_angle" value="0.0" />
    <!--param name="pcap" value="/home/zhangt/Desktop/Ruby-enosh-0421/hi-10m.pcap"/-->
  </node>

  <node  name="cloud_node" pkg="rslidar_pointcloud" type="cloud_node" output="screen" >
    <!--param name="input_packets_topic" value="ruby/rslidar_packets"/-->
    <param name="model" value="$(arg model)"/>
    <param name="angle_path" value="$(arg lidar_param_path)/angle.csv" />
    <param name="channel_path" value="$(arg lidar_param_path)/ChannelNum.csv" />
    <param name="max_distance" value="250"/>
    <param name="min_distance" value="1.0"/>
  </node>

  <node name="rviz" pkg="rviz" type="rviz"  args="-d $(find rslidar_pointcloud)/rviz_cfg/rslidar.rviz" />

</launch>
